### PR TITLE
BIG-24226 update hello world example

### PR DESCRIPTION
**Pull request by gregory**
Just updated the hello world app to highlight the scopes associated to the received token and update the code to reflect the the expected behavior when receiving new token for existing installs.

### CODE REVIEW

**Comment by bc-croh92** [comment]
:+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM bigcommerce/ruby:2.0.0-p576
+FROM hub.services.bigcommerceapp.com/bigcommerce/ruby:2.0.0-p576
```
Comment (mattolson) [inline Dockerfile]: Should we switch to 2.2.3?

File: docker-compose.yml [hunk 2]
```diff
@@ -3,9 +3,8 @@ dev:
     service: hello_world_ruby
     file: ../../docker-compose.yml
   volumes:
-    - .:/opt/services/hello-world-app-ruby-sinatra
+    - .:/opt/services/hello-world-ruby
   command: bash -c "bundle && foreman start"
-  build: .
```
Comment (mattolson) [inline docker-compose.yml]: I find this useful, I don't think we should remove it?

File: hello.rb [hunk 3]
```diff
@@ -42,6 +42,7 @@ class Store
   property :id,           Serial
   property :store_hash,   String, required: true
   property :access_token, String, required: true
+  property :scope,        Text, required: true
```
Comment (mattolson) [inline hello.rb]: Should be String, right? How long could this possibly be?

File: hello.rb [hunk 4]
```diff
@@ -93,15 +94,17 @@ def bc_api_working?
   email = auth[:info][:email]
   store_hash = auth[:extra][:raw_info][:context].split('/')[1]
   token = auth[:credentials][:token].token
+  scope = params[:scope]
```
Comment (mattolson) [inline hello.rb]: Looks like this is available in the omniauth info: https://github.com/bigcommerce/omniauth-bigcommerce/blob/master/lib/omniauth/strategies/bigcommerce.rb#L39

Comment (gregory) [inline hello.rb]: yes, but for some reason, not set properly, yet.

**Comment by mattolson** [comment]
:+1: Please follow up on why the omniauth gem is not delivering the scope as expected.

**Pull request merged**